From: Yajun Deng <yajun.deng@linux.dev>
Subject: mm/mm_init.c: remove redundant pr_info when node is memoryless
Date: Wed, 6 Sep 2023 17:11:13 +0800

There is a similar pr_info in free_area_init_node(), so remove the
redundant pr_info.

Link: https://lkml.kernel.org/r/20230906091113.4029983-1-yajun.deng@linux.dev
Signed-off-by: Yajun Deng <yajun.deng@linux.dev>
Cc: Mike Rapoport (IBM) <rppt@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/mm_init.c |    2 --
 1 file changed, 2 deletions(-)

--- a/mm/mm_init.c~mm-mm_initc-remove-redundant-pr_info-when-node-is-memoryless
+++ a/mm/mm_init.c
@@ -1871,8 +1871,6 @@ void __init free_area_init(unsigned long
 		pg_data_t *pgdat;
 
 		if (!node_online(nid)) {
-			pr_info("Initializing node %d as memoryless\n", nid);
-
 			/* Allocator not initialized yet */
 			pgdat = arch_alloc_nodedata(nid);
 			if (!pgdat)
_
